### PR TITLE
Create paul scanlon 100 days of Gatsby blog post and add  assets

### PR DESCRIPTION
**Pull request by mgienow**
One dev + 100 days = 10 new themes and plugins

### CODE REVIEW

File: docs/blog/2020-04-20-paulie-scanlons-journey-of-100-days/index.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+After a little research I discovered that Gatsby is just React...BUT it still maintains effective Search Engine Optimization by creating static HTML pages with real and actual Google indexable metadata and Open Graph images for each and every page or post!
+
+**When the penny dropped it was an absolute hammer blow!**
+
+With Gatsby you get back all that good stuff we kind of lost when "Websites" became "Single Page Applications" built with React.
+
+When the [#100DaysOfGatsby](https://www.gatsbyjs.org/blog/tags/100-days-of-gatsby/) challenge was first announced I thought, _Yes, please_. I'm totally up for getting involved, but it soon transpired that pretty much every challenge was something I'd already learnt myself. _No matter,_ I thought. I can still continue to develop my React, JavaScript and Gatsby skills and by using the [hashtag](https://twitter.com/hashtag/100DaysOfGatsby) I hoped to share what I'd learnt and to find other folks out there doing the same.
```
Comment (laurieontech) [inline docs/blog/2020-04-20-paulie-scanlons-journey-of-100-days/index.md:27]:
```suggestion
When the [#100DaysOfGatsby](/blog/tags/100-days-of-gatsby/) challenge was first announced I thought, _Yes, please_. I'm totally up for getting involved, but it soon transpired that pretty much every challenge was something I'd already learnt myself. _No matter,_ I thought. I can still continue to develop my React, JavaScript and Gatsby skills and by using the [hashtag](https://twitter.com/hashtag/100DaysOfGatsby) I hoped to share what I'd learnt and to find other folks out there doing the same.
```

**Comment by laurieontech** [comment]
One quick edit but I think this looks good!

**Comment by mgienow** [comment]
I just looked at it and it looks great, ready to merge when you are 👼

**Comment by laurieontech** [comment]
Yup! Looks good!

**Pull request merged**